server/planning/assignments/assignments.py

```diff
@@ -336,6 +338,53 @@ def send_assignment_notification(self, updates, original=None, force=False):
         else:
             event_item = None
 
+        # Create the ICS file to be added to the email usable in google calendar.
+        ical = Calendar()
+        ical.add('BEGIN', 'VCALENDAR')
+        ical.add('PRODID', '-//Superdesk//NTB//EN')
+        ical.add('VERSION', '2.0')
+        ical.add('BEGIN', 'VEVENT')
```

Comment:
The `icalendar` library has an `Event` class that can be used to add subcomponents to the file (https://icalendar.readthedocs.io/en/latest/usage.html#subcomponents).

This removes the need to manually add `BEGIN/END` attributes for the event. It would be better to let the library handle this for us